src/basic_recipes/basic_recipes.jl

```diff
@@ -1192,24 +1192,17 @@ function plot!(p::Spy)
         end
         points, convert(Vector{Float32}, color)
     end
+
     replace_automatic!(p, :colorrange) do
         lift(xycol) do (xy, col)
             extrema_nan(col)
         end
     end
+
     marker = lift(p.marker) do x
-        if x === automatic
-            # If we currently use GLMakie, we can go super fast!
-            BackendModule = parentmodule(typeof(AbstractPlotting.current_backend[]))
-            if nameof(BackendModule) == :GLMakie
-                BackendModule.FastPixel()
-            else
-                :rect
-            end
-        else
-            x
-        end
+        return x === automatic ? FastPixel() : x
```

Comment:
I wonder if FastPixel should take an alpha modifier, so that you can use it to show not just the point but also the density of data within a pixel rect.

Reply:
that can be done via the color, no?

Reply:
I'm not sure how it works.  If I have 2 points in my data which lie within the same pixel rectangle, and `color = (:red, .1)`, is the new alpha of that point now .2?

Reply:
Should be!